objects/authentication_factor.json

```diff
@@ -0,0 +1,82 @@
+{
+  "caption": "Authentication Factor",
+  "description": "The authentication factor object provides information about the factor used to authenticate in an mfa request.",
+  "name": "authentication factor",
+  "extends": "object",
+  "attributes": {
+    "factor_device": {
+      "description": "Device used to complete the authentication request.",
+      "requirement": "recommended"
+    },
+    "factor_type": {
+      "enum": {
+        "1": {
+          "caption": "SMS"
+        },
+        "2 question": {
+          "caption": "Security Question"
+        },
+        "3": {
+          "caption": "Call"
+        },
+        "4": {
+          "caption": "Time-based One-time Password (totp)"
+        },
+        "5": {
+          "caption": "Push Factor"
+        },
+        "6": {
+          "caption": "Token"
+        },
+        "7": {
+          "caption": "Security Key"
+        },
+        "8": {
+          "caption": "Email"
+        },
+        "9": {
+          "caption": "U2F"
+        },
+        "10": {
+          "caption": "WebAuthn"
+        }
+      },
+      "requirement": "required"
+    },
+    "factor_question": {
+      "description": "Question provided for user to answer for question based authentication factor.",
+      "requirement": "optional"
+    },
+    "factor_telephony_num": {
```

Comment:
`question`: are there any other examples of phone number fields in OCSF, and do they abbrev.?  e.g. say `num` or `number`?

Reply:
there are not, I updated this to be number